bin/desi_squeze.py

```diff
@@ -116,28 +117,31 @@ def main():
     candidates = Candidates(mode="operation", model=model,
                             name=args.output_filename)
 
+    """
     # look for candidates
-    userprint("Looking for candidates")
+    userprint('Looking for candidates')
     if save_file is None:
-        candidates.find_candidates(spectra.spectra_list(), save=False)
+        candidates.find_candidates(Spectra.spectra_list(), save=False)
     else:
-        candidates.find_candidates(spectra.spectra_list(), save=True)
+        candidates.find_candidates(Spectra.spectra_list(), save=True)
+    """
+    candidates.find_candidates(squeze_spectra.spectra_list(), save=False)
 
     # compute probabilities
     userprint("Computing probabilities")
-    candidates.classify_candidates()
+    candidates.classify_candidates(save=False)
 
     # filter results
     data_frame = candidates.candidates()
     data_frame = data_frame[~data_frame["DUPLICATED"]][args.keep_cols]
 
-    # save results
-    hdu = fits.BinTableHDU.from_columns([fits.Column(name=col,
-                                                     format=convert_dtype(dtype),
-                                                     array=data_frame[col])
-                                         for col, dtype in zip(data_frame.columns,
-                                                               data_frame.dtypes)])
-    hdu.writeto(args.output_filename, overwrite=True)
+    data_out = np.zeros(len(data_frame), dtype=[('TARGETID','int64'),('Z_SQ','float64'),('Z_SQ_CONF','float64')])
```

Comment:
You have modifiied the way data is saved. This means that now the args.keep_cols is useless. We should remove it then

Reply:
args.keep-cols is now removed